.gitignore

```diff
@@ -3,4 +3,4 @@ dist
 extension/background.js
 extension/content_script.js
 extension/overview/overview.js
-
+extension/options/options.js
```

Comment:
Could you `rebase -i` the commits and `fixup` this commit to glue it into the one before it? Otherwise this repo will keep carrying 2MB of unused code around for ever.